### PR TITLE
chore: prepare for launch

### PR DESCRIPTION
**Pull request by nicole-obrien**
## Summary

> Please summarize your changes, describing __what__ they are and __why__ they were made.

...

## Changelog

```
- Please write a list of changes
```

## Relevant Issues

> Please list any related issues using [development keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

...

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
  - [ ] MacOS
  - [ ] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by cvarley100** [comment]
🚀

File: README.md [hunk 1]
```diff
@@ -27,19 +27,22 @@
...
-- Build from source code (see [desktop README](https://github.com/iotaledger/firefly/blob/develop/packages/desktop/README.md))
+-   Download from [official Firefly Shimmer website](https://firefly.iota.org)
+-   Download from the latest [GitHub release](https://github.com/iotaledger/firefly/releases/tag/desktop-2.0.0)
+-   Build from source code (see our handbook guide for ["Environment Setup"](https://iotaledger.github.io/firefly/guides/environment-setup/))
 
 ## Contributing
 
 Help out the Firefly project by filing a [bug report](https://github.com/iotaledger/firefly/issues/new?assignees=&labels=bug&template=bug_report.yml), making a [feature request](https://github.com/iotaledger/firefly/issues/new?assignees=&labels=feat&template=feature_request.md) or opening a [pull request](https://github.com/iotaledger/firefly/pulls/).
-See the [contribution guidelines](https://github.com/iotaledger/firefly/blob/develop/.github/CONTRIBUTING.md) for more.
+If making a code contribution, be sure to checkout the [contribution guidelines](https://github.com/iotaledger/firefly/blob/stardust-develop/.github/CONTRIBUTING.md) beforehand.
```
Comment (Tuditi) [inline README.md:43]:
```suggestion
Checkout the [contribution guidelines](https://github.com/iotaledger/firefly/blob/stardust-develop/.github/CONTRIBUTING.md) before making a code contribution.
```

File: README.md [hunk 2]
```diff
@@ -27,19 +27,22 @@
...
+
+For all of the details, see our [development handbook](https://iotaledger.github.io/firefly/).
```
Comment (Tuditi) [inline README.md:45]:
```suggestion
See our [development handbook](https://iotaledger.github.io/firefly/) for more details.
```

File: packages/desktop/App.svelte [hunk 3]
```diff
@@ -90,10 +82,10 @@
 
         /* eslint-disable no-undef */
         // @ts-expect-error: This value is replaced by Webpack DefinePlugin
-        if (!devMode && get(appStage) === AppStage.PROD) {
-            await setAppVersionDetails()
-            pollCheckForAppUpdate()
-        }
+        // if (!devMode && get(appStage) === AppStage.PROD) {
+        //     await setAppVersionDetails()
```
Comment (Tuditi) [inline packages/desktop/App.svelte:86]: I think you should only uncomment the polling if it doesn't work

**Review by Tuditi — CHANGES_REQUESTED** [review]
Looks good! I think we have to uncomment setAppVersionDetails otherwise the menu won't be properly set

File: packages/desktop/App.svelte (hunk 3 above)
Comment (nicole-obrien) [reply packages/desktop/App.svelte:86]: The setAppVersionDetails is throwing an error too causing the whole function to cancel and not register electron events.

File: README.md (hunk 1 above)
Comment (nicole-obrien) [reply README.md:43]: Suggest these to matt, this isn't even visible unless in the branch.

File: README.md (hunk 2 above)
Comment (nicole-obrien) [reply README.md:45]: Suggest these to matt, this isn't even visible unless in the branch.

Comment (Tuditi) [reply README.md]: I pushed the changes

**Review by Tuditi — APPROVED** [review]
All good from my side!

**Pull request merged**